lms/djangoapps/instructor_task/api.py

```diff
@@ -63,7 +69,23 @@ def get_running_instructor_tasks(course_id):
     # exclude states that are "ready" (i.e. not "running", e.g. failure, success, revoked):
     for state in READY_STATES:
         instructor_tasks = instructor_tasks.exclude(task_state=state)
-    return instructor_tasks.order_by('-id')
+
+    instructor_tasks = instructor_tasks.order_by('-id')
+
+    # list down remote grade book tasks
+    now = datetime.datetime.now()
+    rgb_tasks = InstructorTask.objects.filter(
+        course_id=course_id,
+        task_state__icontains="success",
+        task_type=TASK_TYPE_EXPORT_GRADES_TO_RGB,
+        updated__lte=now,
+        updated__gte=now - datetime.timedelta(days=5)
+    ).order_by('-id')
```

Comment:
Let's just limit this to 3 results instead of showing the last 5 days, and also order by 'updated' (descending): `rgb_tasks = InstructorTask.objects.filter(...).order_by('-updated')[0:3]`